packages/web/package.json

```diff
@@ -230,13 +230,13 @@
     "eslint-config-prettier-standard": "4.0.1",
     "eslint-config-standard": "14.1.1",
     "eslint-import-resolver-alias": "1.1.2",
-    "eslint-plugin-import": "2.20.2",
+    "eslint-plugin-import": "2.24.2",
```

Comment:
(all these changes make the eslint versions of things the same as mobile, not strictly necessary but thought it would make things less brittle for now before we transition to shared config)